docs/views.py

```diff
@@ -89,7 +90,12 @@ def serve_docs(request, path, **kwargs):
         raise DocsRootSettingError('DOCS_ROOT setting value is incorrect: %s (must be a valid path)' % DOCS_ROOT)
     if 'document_root' not in kwargs:
         kwargs['document_root'] = DOCS_ROOT
-    return serve(request, path, **kwargs)
+    
+    try:
```

Comment:
Are you sure that `path` always ends with `/`?

Reply:
In my limited testing it always seemed to be the case, but it seems unwise to assume. Maybe something like this would be a better idea?

```python
from os.path import join
path = join(path, "index.html")
```

Let me know if you want me to make that change.

Reply:
I ended up making that `os.path.join` change in https://github.com/adamghill/django-docs/commit/a68af62786589bc381d81a165cd8344519931526. I also added a setting for `DOCS_DIRHTML` so that I could disable the permanent redirect for `DocsRootView` since that is not needed when building with the `dirhtml` builder.